_data/sites.json

```
@@ -2983,6 +2983,17 @@
         ]
     },
 
+    {
+        "name": "DaFONT",
+        "url": "https://www.dafont.com/forum/read/12643/how-do-i-delete-my-account",
+        "difficulty": "hard",
+        "notes": "You'll send a private message to Rodolphe : http://www.dafont.com/fr/profile.php?user=1 (Wrote by Vinz).",
+        "notes_it": "Dovrai mandare un messagio privato a Rodolphe http://www.dafont.com/fr/profile.php?user=1 (Scritto da Vinz): [scritto da madrelingua / wrote by a motherlanguage]",
```

I'm confused as to whether the note of motherlanguage here is a tip for the user or a note to us, the jdm maintainers. Can you clarify your intent, please?

---

I wrote this to let the italian justdeleteme users that the translation is 100% accurate

---

I see, there's no need for this note, but thanks for letting us know.
```suggestion
        "notes_it": "Dovrai mandare un messagio privato a Rodolphe: <a href='http://www.dafont.com/fr/profile.php?user=1'>",
```